CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ All notable changes to this project will be documented in this file. If a contri
 
 ## Unreleased
 
-- n/a
+- Fix issue with multiple iframes and StyleSheetManager instances. @darthtrevino (see [#1634])
```

Comment:
Can we make this generic as well like the test?

I.e. fix last className caching strategy to take StyleSheet cache into account

Or sth